launch-game.cmd

```diff
@@ -5,18 +5,18 @@ title OpenRA
 FOR /F "tokens=1,2 delims==" %%A IN (mod.config) DO (set %%A=%%B)
 if exist user.config (FOR /F "tokens=1,2 delims==" %%A IN (user.config) DO (set %%A=%%B))
 set TEMPLATE_LAUNCHER=%0
-set MOD_SEARCH_PATHS=%~dp0mods,./mods
+set MOD_SEARCH_PATHS=%~dp0mods,../mods
 
 if "!MOD_ID!" == "" goto badconfig
 if "!ENGINE_VERSION!" == "" goto badconfig
 if "!ENGINE_DIRECTORY!" == "" goto badconfig
 
 set TEMPLATE_DIR=%CD%
-if not exist %ENGINE_DIRECTORY%\OpenRA.Game.exe goto noengine
+if not exist %ENGINE_DIRECTORY%\bin\OpenRA.Game.exe goto noengine
 >nul find %ENGINE_VERSION% %ENGINE_DIRECTORY%\VERSION || goto noengine
 cd %ENGINE_DIRECTORY%
 
-OpenRA.Game.exe Game.Mod=%MOD_ID% Engine.LaunchPath="%TEMPLATE_LAUNCHER%" "Engine.ModSearchPaths=%MOD_SEARCH_PATHS%"  "%*"
+bin\OpenRA.Game.exe Game.Mod=%MOD_ID% Engine.LaunchPath="%TEMPLATE_LAUNCHER%" "Engine.ModSearchPaths=%MOD_SEARCH_PATHS%"  "%*"

```

Comment:
This should be: `bin\OpenRA.exe Game.Mod=%MOD_ID% Engine.EngineDir=".." Engine.LaunchPath="%TEMPLATE_LAUNCHER%" "Engine.ModSearchPaths=%MOD_SEARCH_PATHS%"  "%*"`

Reply:
Thanks for all the feedback on the launcher. I wasn't aware of the https://github.com/OpenRA/OpenRAModSDK project. I will merge the changes there since the tag 20200503 (with the exception of the last commit for the playtest that came out the other day). That should hopefully solve all of these more systematically than what I was doing.

Reply:
Not a problem, I'll continue to track this and test on Windows.
I've been looking to get into helping with this project, and your work has been the kick in the pants I needed.